doc/reference/views.rst

```diff
@@ -1934,6 +1935,44 @@ assuming ``foo`` is a field and ``bar`` is a filter an action context of:
 will automatically enable the ``bar`` filter and search the ``foo`` field for
 *acro*.
 
+.. _reference/views/map:
+
+Map
+===
+
+This view is able to display records on a map and the routes between them. The record are represented by pins. It also allows the visualization of fields from the model in a popup tied to the record's pin. The model on which the view is applied should contains a res.partner many2one since the view relies on the res.partner's address and coordinates fields to localize the records.
```

Comment:
"The model on which the view is applied should contains a res.partner many2one since the view relies on the res.partner's address and coordinates fields to localize the records." add emphasis